router/router.nginx.conf.in

```diff
@@ -6,24 +6,24 @@ server {
     }
 
     location / {
-        proxy_pass http://site/;
+        proxy_pass https://site/;
         include /etc/nginx/proxy.conf;
     }
 
-    listen 80 default_server;
-    listen [::]:80 default_server;
+    listen 443 ssl default_server;
+    listen [::]:443 ssl default_server;
 }
 
 server {
     server_name scorecard.*;
 
     location / {
-        proxy_pass http://scorecard/;
+        proxy_pass https://scorecard/;
         include /etc/nginx/proxy.conf;
     }
 
-    listen 80;
-    listen [::]:80;
+    listen 443 tls;
```

Comment:
These should be ssl I believe: http://nginx.org/en/docs/http/ngx_http_core_module.html

Syntax: | listen port [default_server] [ssl] [http2 \| spdy]  ...
Syntax: | listen address[:port] [default_server] [ssl] [http2 \| spdy]  ...

Reply:
indeed